src/main/java/desktop/pages/AccountPage.java

```diff
@@ -2,71 +2,55 @@
 
 import abstractclasses.page.AbstractPage;
 import desktop.fragments.NavigationBarFragment;
+import driver.SingletonDriver;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 
-public class AccountPage extends AbstractPage {
+import static constants.Constants.SIGN_IN_PAGE_URL;
+
 
+public class AccountPage extends AbstractPage {
+    @FindBy(xpath = "//h1[contains(@class,'a-spacing-top-small')]")
+    public WebElement titleText;
     @FindBy(xpath = "//*[@id='ap_email']")
     WebElement emailField;
-
-    @FindBy(xpath = "//div[contains(@class,'a-input-text-group')]//input[@type='password']")
+    @FindBy(xpath = "//input[@type='password']")
     WebElement passwordField;
-
-    @FindBy(xpath = "//form[@name='signIn']//input[@id='signInSubmit']")
+    @FindBy(xpath = "//input[@id='signInSubmit']")
     WebElement submitButton;
-
-    @FindBy(xpath = "//h1[contains(@class,'a-spacing-top-small')]")
-    public WebElement titleText;
+    NavigationBarFragment navigationBar = new NavigationBarFragment(driver);
 
     public AccountPage(WebDriver driver) {
         super(driver);
-    }
-
-    NavigationBarFragment navigationBar = new NavigationBarFragment(driver);
-
-    private static String accountPageUrl = "https://www.bookdepository.com/account/login/to/account";
-    private static String successfulLoginPageURL = "https://www.bookdepository.com/ap/signin";
-
-
-    public void openPage() {
-        getDriver().get(accountPageUrl);
+        SingletonDriver.getInstance();
     }
 
     public boolean checkURL() {
-        String currentURL = driver.getCurrentUrl();
-        return (currentURL == successfulLoginPageURL);
+        return SingletonDriver.getInstance().getCurrentUrl().equals(SIGN_IN_PAGE_URL);
     }
 
-    public void enterEmail(String strEmail) {
+    public void enterEmail(String email) {
         emailField.clear();
-        emailField.sendKeys(strEmail);
+        emailField.sendKeys(email);
     }
 
-    public void enterPassword(String strPassword) {
-        passwordField.sendKeys(strPassword);
+    public void enterPassword(String password) {
+        passwordField.sendKeys(password);
     }
 
-    public void clickSubmitButton() {
+    public void clickSignInButton() {
         submitButton.click();
     }
 
     public Boolean checkLoginTitleDisplayed() {
```

Comment:
the same here - method should be renamed to IsLoginTitleDisplayed() or add the assertion inside

Reply:
fixed

Reply:
sorry, my bad: please rename method according to java naming conventions - isLoginTitleDisplayed()

Reply:
resolved